ui/lib/core/addon/components/form-field.hbs

```diff
@@ -46,7 +46,13 @@
               </option>
             {{/if}}
             {{#each (path-or-array @attr.options.possibleValues @model) as |val|}}
-              <option selected={{eq (get @model this.valuePath) (or val.value val)}} value={{or val.value val}}>
+              <option
+                selected={{eq
+                  (number-to-string (get @model this.valuePath) options=@attr.options.possibleValues)
+                  (or val.value val)
+                }}
```

Comment:
What if instead of the `number-to-string` helper you created a `loose-eq` helper that uses `==` instead of `===` and use that in place of the `eq` helper?

Reply:
Haha! Like I had mentioned this morning? I was worried it was too loosey-goosey, but it does feel much cleaner than transforming types

Reply:
I didn't realize when we were discussing that it was only used to evaluate whether or not the item is selected. In that case I think loose equality is fine. Maybe some comments to indicate that it should only used sparingly for edge cases would be beneficial too.